pyproject.toml

```
@@ -1,3 +1,85 @@
+[build-system]
+requires = ["setuptools>=63.0.0", "wheel"]
+build-backend = "setuptools.build_meta" 
+
+
+[tools.setuptools]
+# Default value is true.
+include-package-data = true
+
+[tool.setuptools.packages.find]
+where = ["mappymatch"]
+
+[project]
+name = "mappymatch"
+version = "0.2.2"
+description = "Package for mapmatching."
+readme = "README.md"
+authors = [{ name = "National Renewable Energy Laboratory" }]
+license = { text = "BSD 3-Clause License Copyright (c) 2022, Alliance for Sustainable Energy, LLC" }
+classifiers = [
+    "Development Status :: 3 - Alpha",
+    "Intended Audience :: Science/Research",
+    "License :: Other/Proprietary License",
+    "Operating System :: OS Independent",
+    "Programming Language :: Python",
+    "Programming Language :: Python :: 3.8",
+    "Topic :: Scientific/Engineering",
+]
+keywords = ["GPS", "map", "match"]
+dependencies = [
+    "geopandas",
+    "shapely",
+    "rtree",
+    "pyproj",
+    "pandas",
+    "numpy",
+    "matplotlib",
+    "osmnx", 
+    "networkx",
+    "folium",
+    "requests",
+]
+requires-python = ">=3.8"
+
+[project.optional-dependencies]
+# Used to run CI. 
+tests = [
+  "black",
+  "isort",
+  "mypy",
+  "types-requests",
+  "flake8",
+  "pytest",
+]
+# Used to build the docs.
+docs = [
+  "sphinx==4.5.0",
+  "sphinx_rtd_theme==1.0.0",
+  "sphinxemoji==0.2.0",
+]
+# Tests + docs + other.
+dev = [
+  "black",
+  "coverage",
+  "interrogate",
+  "isort",
+  "mypy",
+  "pre-commit",
+  "pytest",
+  "sphinx==4.5.0",
+  "sphinx_rtd_theme==1.0.0",
+  "sphinxemoji==0.2.0",
+  ]
+release = [
+  "build",
+  "twine"
+]
```

How would you feel about consolidating the release block into the dev block for simplicity? And, how would you feel about just referring to the tests and docs dependencies blocks? I've tested the below locally and it seems to be working.

```suggestion
dev = [
  "mappymatch[tests]",
  "mappymatch[docs]",
  "coverage",
  "interrogate",
  "pre-commit",
  "build",
  "twine",
]
```